xt/02_perlcritic.t

```diff
@@ -15,7 +15,7 @@ eval {
 note $@ if $@;
 plan skip_all => "Perl::Critic 1.105+ or Test::Perl::Critic 1.02+ is not installed." if $@;
 
-all_critic_ok('lib', 'script', 'bin');
```

Comment:
why did you skip these directories?

Reply:
Because they don't exist, and Test::Perl::Critic (when run from `minil dist`) displays a warning without this change:


```
t/14_listen.t .................. ok
xt/02_perlcritic.t ............. Can't stat script: No such file or directory
 at /usr/share/perl5/vendor_perl/Perl/Critic/Utils.pm line 1101.
Can't stat bin: No such file or directory
 at /usr/share/perl5/vendor_perl/Perl/Critic/Utils.pm line 1101.
xt/02_perlcritic.t ............. ok
xt/04_dependents.t ............. skipped: AUTHOR_TESTING is required.
xt/author/11_net_emptyport.t ... 1/11 # Port: 5001 - tcp
# New port: 5002 - tcp
# Port: 5001 - udp
# New port: 5002 - udp
xt/author/11_net_emptyport.t ... ok
```

Reply:
ah, i see.